mypy/newsemanal/semanal.py

```diff
@@ -222,6 +222,7 @@ class NewSemanticAnalyzer(NodeVisitor[None],
     # not be found in phase 1, for example due to * imports.
     errors = None  # type: Errors     # Keeps track of generated errors
     plugin = None  # type: Plugin     # Mypy plugin for special casing of library features
+    statement = None  # type: Node    # Statement/definition being analyzed
```

Comment:
This is probably bikesheding, but I would call this `cur_statement`, similar to `cur_mod_id` above.

Also, should the type be actually `Statement`?